week-2/F-logical-operators/exercise2.js

```diff
@@ -5,8 +5,19 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
-
+function isNegative(num) {
+  return num < 0 ;
+}
+function isBetween5and10(num) {
+  return num >= 5 && num <= 10;
+}
+function isLessThan8Characters(name) {
+  return name.length < 8;
+}
+function startsWithD(letter) {
+  letter = "D";
+  return true;
```

Comment:
This function always returns `true`. it should test whether the name passed in starts with `D` so the parameter shouldn't be called `letter`, which implies it is a single character rather than a string.

Reply:
Ok Luke, I changed to name instead and updated